interchain-exchange/tutorial/05-order-initialization.md

```diff
@@ -0,0 +1,356 @@
+
+// BankKeeper defines the expected bank keeper
+type BankKeeper interface {
+	SendCoins(ctx sdk.Context, fromAddr sdk.AccAddress, toAddr sdk.AccAddress, amt sdk.Coins) error
+	MintCoins(ctx sdk.Context, moduleName string, amt sdk.Coins) error
+	BurnCoins(ctx sdk.Context, moduleName string, amt sdk.Coins) error
+	SendCoinsFromModuleToAccount(ctx sdk.Context, senderModule string, recipientAddr sdk.AccAddress, amt sdk.Coins) error
+	SendCoinsFromAccountToModule(ctx sdk.Context, senderAddr sdk.AccAddress, recipientModule string, amt sdk.Coins) error
+}
+```
+
+Next, in the `keeper` directory, specify the bank so that you can access it in your module. 
+
+```go
+// x/ibcdex/keeper/keeper.go
+type (
+	Keeper struct {
+		cdc           codec.Marshaler
+		storeKey      sdk.StoreKey
+		memKey        sdk.StoreKey
+		channelKeeper types.ChannelKeeper
+		portKeeper    types.PortKeeper
+		scopedKeeper  capabilitykeeper.ScopedKeeper
+		bankKeeper    types.BankKeeper
+	}
+)
+
+func NewKeeper(
+	cdc codec.Marshaler,
+	storeKey,
+	memKey sdk.StoreKey,
+	channelKeeper types.ChannelKeeper,
+	portKeeper types.PortKeeper,
+	scopedKeeper capabilitykeeper.ScopedKeeper,
+	bankKeeper types.BankKeeper,
+) *Keeper {
+	return &Keeper{
+		cdc:           cdc,
+		storeKey:      storeKey,
+		memKey:        memKey,
+		channelKeeper: channelKeeper,
+		portKeeper:    portKeeper,
+		scopedKeeper:  scopedKeeper,
+		bankKeeper:    bankKeeper,
+	}
+}
+```
+
+Lastly, the `app.go` file that describes which modules are used in the blockchain application, add the bank keeper to the `ibcdexKeeper`
+
+```go
+// app/app.go
+app.ibcdexKeeper = *ibcdexkeeper.NewKeeper(
+  appCodec,
+	keys[ibcdextypes.StoreKey],
+	keys[ibcdextypes.MemStoreKey],
+	app.IBCKeeper.ChannelKeeper,
+	&app.IBCKeeper.PortKeeper,
+	scopedIbcdexKeeper,
+	app.BankKeeper,
+)
+```
+
+The app will mint and burn token using the bank account. The use this feature, the module must have a _module account_. To enable the _module account_ you declare this permission in the _module account permissions_ structure of the auth module. This is done in `app.go` file.
+
+```go
+// app/app.go
+maccPerms = map[string][]string{
+    authtypes.FeeCollectorName:     nil,
+    distrtypes.ModuleName:          nil,
+    minttypes.ModuleName:           {authtypes.Minter},
+    stakingtypes.BondedPoolName:    {authtypes.Burner, authtypes.Staking},
+    stakingtypes.NotBondedPoolName: {authtypes.Burner, authtypes.Staking},
+    govtypes.ModuleName:            {authtypes.Burner},
+    ibctransfertypes.ModuleName:    {authtypes.Minter, authtypes.Burner},
+    ibcdextypes.ModuleName: 		{authtypes.Minter, authtypes.Burner}, // <--
+}
+```
+
+## Ensure No Native Token are Minted
+
+Ensure that no new native tokens are minted in case the module is connected to a faulty blockchain. Rename this functionality into lock and unlock and reuse functions from `ibc-transfer` module with an escrow address to ensure no new token creation.
```

Comment:
```suggestion
Ensure that no new native tokens are minted in case the module is connected to a malicious or faulty counterparty. Rename this functionality into lock and unlock and reuse functions from `ibc-transfer` module with an escrow address to ensure no new token creation.
```